src/sidebar/components/MarkdownView.tsx

```diff
@@ -35,21 +76,49 @@ export default function MarkdownView({
     });
   }, [markdown]);
 
+  useEffect(() => {
+    const listenerCollection = new ListenerCollection();
+    const foundMentions = replaceMentionTags(content.current!, mentions);
+
+    for (const [mention, element] of foundMentions) {
+      listenerCollection.add(element, 'mouseenter', () => {
+        setActiveMention(mention);
+        mentionsPopoverAnchorRef.current = element;
+      });
+      listenerCollection.add(element, 'mouseleave', () => {
+        setActiveMention(null);
+        mentionsPopoverAnchorRef.current = null;
+      });
+    }
+
+    return () => listenerCollection.removeAll();
+  }, [html, mentions]);
+
   // NB: The following could be implemented by setting attribute props directly
   // on `StyledText` (which renders a `div` itself), versus introducing a child
   // `div` as is done here. However, in initial testing, this interfered with
   // some overflow calculations in the `Excerpt` element. This could be worth
   // a review in the future.
   return (
     <div className="w-full break-anywhere cursor-text">
-      <StyledText>
+      <StyledText classes="relative">
```

Comment:
A `relative` wrapper is needed for when the native Popover API is not supported.

Reply:
Can you make sure this is commented.